tests/test_utils.py

```diff
@@ -73,3 +73,23 @@ def test_get_normalized_values(values, expected_result):
         np.testing.assert_almost_equal(sum(normalized_values), 1.0, 6)
     else:
         np.testing.assert_almost_equal(sum(normalized_values), 0.0, 6)
+
+
+def test_regularize_to_error_weighted_mean_neighbors():
+    X = pd.DataFrame({"a": [0, 1], "b": [3, 6], "c": [2, 7]})
```

Comment:
Why are these lines part of the test?
```
    X = pd.DataFrame({"a": [0, 1], "b": [3, 6], "c": [2, 7]})
    features = utils.get_feature_column_names(X)
    np.testing.assert_equal(features, ["a", "b", "c"])
    features = utils.get_feature_column_names(X, exclude_columns=["a"])
    np.testing.assert_equal(features, ["b", "c"])
```
It seems like there are two functions `get_feature_column_names` and `regularize_to_error_weighted_mean_neighbors` being tested here.